requirements.txt

```diff
@@ -4,3 +4,6 @@ chromadb==0.3.22
 llama-cpp-python==0.1.48
 urllib3==1.26.6
 pdfminer.six==20221105
+unstructured==0.6.6
+extract-msg==0.41.1
+pandocs
```

Comment:
I think it'd be good to fix the version. Latest is 2.3

Reply:
It is a point of view, personnally I think it's important for the direct dependencies.  I've had an experience with version conflicts between platforms, one platform supportint version X.Y and the other X.Z.  I am setting it to 2.3 though.

Reply:
While I agree with you generally, the space is moving so fast that breaking changes are deployed everyday. Fixing versions may help with that situation